### PR TITLE
Cu 21gqyyq refactor search result sorter and refactor map view

### PR DESCRIPTION
**Pull request by cg-2611**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,9 +0,0 @@
-{
-  "recommendations": [
```
Comment (bridges-wood) [inline .vscode/extensions.json:2]: Why was this deleted?

File: packages/client/src/App.tsx [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+    }
+  }`
+
+export type LocationType = Pick<Location, 'latitude' | 'longitude' | 'id'>;
+
+export interface LocationTypeWithRating extends LocationType {
+  business : Pick<Business, 'sustainabilityScore'>
+}
 
 const ExampleBusiness : DisplayableBusiness =  {
```
Comment (bridges-wood) [inline packages/client/src/App.tsx:34]: Do we need this anymore?

File: packages/client/src/App.tsx [hunk 3]
```diff
@@ -45,7 +62,7 @@ export const FeedScreen = ({ navigation } : Props) => {
   return (
     <ApolloProvider client={client}>
       <View style={styles.container}>
-        <MapView showRadius />
+        {/* <MapView showRadius /> */}
```
Comment (bridges-wood) [inline packages/client/src/App.tsx:65]: Why is this commented out?

File: packages/client/src/App.tsx [hunk 4]
```diff
@@ -76,14 +93,41 @@ export const AppNavigator = () => {
...
 
 const App = () => {
+  const [distance, setDistance] = useState<Option>(DISTANCES[INITIAL_DISTANCE_INDEX])
+  const { data } = useQuery<{ locations : LocationTypeWithRating[]}>(GET_COORDINATES)
+
   return (
-    <NavigationContainer>
-      <AppNavigator />
-    </NavigationContainer>
+    // <NavigationContainer>
```
Comment (bridges-wood) [inline packages/client/src/App.tsx:100]: Why is this commented out?

File: packages/client/src/cache.ts [hunk 5]
```diff
@@ -1,7 +1,6 @@
 import { InMemoryCache, makeVar } from '@apollo/client'
-import { SortOptions } from '@futureproof/typings'
 
 const cache = new InMemoryCache()
-export const sortOption = makeVar<SortOptions>('name_asc')
```
Comment (bridges-wood) [inline packages/client/src/cache.ts:5]: Why has this type changed?

File: packages/client/src/components/MapView.spec.tsx [hunk 6]
```diff
@@ -42,6 +42,8 @@ const deniedPermissions : LocationPermissionResponse = {
 
 let location : MockedObject<typeof Location>
 
+const distances = [100, 250, 500, 1000, 1500, 2000]
```
Comment (bridges-wood) [inline packages/client/src/components/MapView.spec.tsx:45]: Why is this here if it is unused?

**Review by bridges-wood — CHANGES_REQUESTED** [review]
You need to run the test suite.

File: .vscode/extensions.json (hunk 1 above)
Comment (cg-2611) [reply .vscode/extensions.json:2]: did not mean to commit that, ill add it back

File: packages/client/src/App.tsx (hunk 2 above)
Comment (cg-2611) [reply packages/client/src/App.tsx:34]: not sure i just moved it when refactoring

File: packages/client/src/App.tsx (hunk 3 above)
Comment (cg-2611) [reply packages/client/src/App.tsx:65]: did not mean to push these changes, they were for debugging, ill fix this

File: packages/client/src/cache.ts (hunk 5 above)
Comment (cg-2611) [reply packages/client/src/cache.ts:5]: SortOptions type doesnt exist anymore

File: packages/client/src/components/MapView.spec.tsx (hunk 6 above)
Comment (cg-2611) [reply packages/client/src/components/MapView.spec.tsx:45]: it was used in tests that have now been removed, ill remove this

**Review by bridges-wood — APPROVED** [review]
